roles/kubernetes-apps/policy_controller/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+# Limits for calico apps
```

Comment:
IMHO roles/kubernetes-apps/calico_policy_controller/defaults/main.yml is better as there is weave network policy controller also. It might create a confusion in future.